src/Container.php

```diff
@@ -21,6 +21,11 @@ class Container implements ContainerInterface, ArrayAccess
     protected $items = [];
 
     /**
+     * @var array - \League\Container\ServiceProvider[]
```

Comment:
DocBlock syntax accepts \League\Container\ServiceProvider[] as valid. You don’t need to put the array of style stuff into the description, thats just a weird hack.

Even if you did, you wouldn’t need the `-`